presto/dbapi.py

```diff
@@ -22,7 +22,16 @@
 from __future__ import print_function
```

Comment:
Since we are using `?` as the parameter style, we should add a module level variable `paramstyle` set to `?` to ensure we comply with the DB-API (since most tools assume the default to be `%(name)` style parameters).

See https://www.python.org/dev/peps/pep-0249/#paramstyle

Reply:
@hashhar Are you suggesting adding `paramstyle = '?'` as a  module variable/constant so that people can reference it when writing queries?

Reply:
Yes. Both for users writing queries and external tools that generate queries (I've seen some for PostgreSQL).

Reply:
Perfect. I'll make that change.

Reply:
Done